lib/cc/engine/rubocop.rb

```diff
@@ -83,17 +85,25 @@ def violation_positions(location)
       end
 
       def violation_json(violation, local_path)
-        {
-          type: "Issue",
-          check_name: "Rubocop/#{violation.cop_name}",
-          description: violation.message,
-          categories: [category(violation.cop_name)],
-          remediation_points: 50_000,
-          location: {
-            path: local_path,
-            positions: violation_positions(violation.location),
-          },
-        }.to_json
+        violation_hash = {
+                            type: "Issue",
+                            check_name: "Rubocop/#{violation.cop_name}",
+                            description: violation.message,
+                            categories: [category(violation.cop_name)],
+                            remediation_points: 50_000,
+                            location: {
+                              path: local_path,
+                              positions: violation_positions(violation.location),
+                            },
+                         }
```

Comment:
Can you format this as:

```rb
violation = {
  type: ...,
  check_name: ...,
}
```

That's our style and will make the diff clearer too.